bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2BridgeHandler.java

```diff
@@ -775,4 +819,92 @@ private void updateThingsScheduled(int delayMilliSeconds) {
             scheduledUpdateTask = scheduler.schedule(() -> updateThingsNow(), delayMilliSeconds, TimeUnit.MILLISECONDS);
         }
     }
+
+    /**
+     * Load the set of automation script ids.
+     */
+    private void loadAutomationScriptIds() {
+        try {
+            automationScriptIds.clear();
```

Comment:
Do we need mutex here, since `automationScriptIds` is a non-threadsafe `HashSet`?

Reply:
Yes, I will add that.